packages/feedback-service/src/config/config.ts

```diff
@@ -0,0 +1,538 @@
+const nodemailer = require("nodemailer");
```

Comment:
Why is this file here? Remove this file immediately. This has all the credentials and secrets. Don't just blindly copy anything.

Reply:
And also remove it from the commit history.

Reply:
@anjneeksharma Remove this file ASAP! This is a security issue. This is a public repository!

Reply:
@anjneeksharma can you please remove this file from repo and commit history asap. Noone should be seeing this file.